pkg/exchange/coinbase/api/v1/get_candles_request.go

```diff
@@ -0,0 +1,38 @@
+package coinbase
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/requestgen"
+)
+
+type Candle struct {
+	Time   time.Time        `json:"time"`
+	Low    fixedpoint.Value `json:"low"`
+	High   fixedpoint.Value `json:"high"`
+	Open   fixedpoint.Value `json:"open"`
+	Close  fixedpoint.Value `json:"close"`
+	Volume fixedpoint.Value `json:"volume"`
+}
+
+type GetCandlesResponse []Candle
+
+// https://docs.cdp.coinbase.com/exchange/reference/exchangerestapi_getproductcandles
+//
+//go:generate requestgen -method GET -url /products/:product_id/candles -type GetCandlesRequest -responseType .GetCandlesResponse
+type GetCandlesRequest struct {
+	client requestgen.AuthenticatedAPIClient
+
+	productID   string  `param:"product_id,required"`
+	granularity *string `param:"granularity" validValues:"60,300,900,3600,21600,86400"`
+	start       *string `param:"start"`
+	end         *string `param:"end"`
```

Comment:
check this file for the timestamp parsing usage:

https://github.com/c9s/bbgo/blob/main/pkg/exchange/max/maxapi/get_k_lines_request.go#L22

Reply:
Got it.

Reply:
Done.